src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -815,6 +815,10 @@
   - name: Create a deployment at resource group scope from a bicep template and a bicepparam parameter file
     text: >
         az deployment group create --resource-group testrg --template-file azuredeploy.bicep --parameters parameters.bicepparam
```

Comment:
Is there a parameter `--name` missing here?

Reply:
In fact, `--name` is not a required parameter, and it has default generation logic when the user does not input it.
https://github.com/Azure/azure-cli/blob/dffb4519329961a65e06098c900df38146899eb3/src/azure-cli/azure/cli/command_modules/resource/_validators.py#L45-L65

Reply:
Got it. Thanks.